defaults/main.yml

```diff
@@ -82,7 +82,4 @@ OnAccessDisableDDD: "true"
 OnAccessExtraScanning: "true"
 OnAccessMountPath:
  - name: /home
- - name: /var
- - name: /var/lib/docker
```

Comment:
Are we sure we want to leave /var as a completely unscanned dir? Seems like it might be a little insecure :/